crates/rpc/rpc/src/eth/filter.rs

```diff
@@ -425,8 +587,14 @@ where
                     .flatten();
                 let (from_block_number, to_block_number) =
                     logs_utils::get_filter_block_range(from, to, start_block, info);
-                self.get_logs_in_block_range(&filter, from_block_number, to_block_number, info)
-                    .await
+                self.get_logs_in_block_range(
+                    &filter,
+                    from_block_number,
+                    to_block_number,
+                    info,
+                    reorg_blocks,
+                )
+                .await
```

Comment:
I don't fully understand why we need to provide the reorged logs here as an argument.

aren't those the logs we already matched then these should always be included in the result